src-backbone/app/locales/en/translation.json

```diff
@@ -0,0 +1,107 @@
+{
+
+    // Sign up
```

Comment:
Technically no. The preprocessor strips these comments out.

These translation files cannot be used by other apps unless they also strip the comment out first.